ert_shared/dark_storage/endpoints/records.py

```diff
@@ -252,10 +254,19 @@ def get_ensemble_responses(
     response_map: Mapping[str, js.RecordOut] = {}
     for response_name in get_response_names():
         obs_keys = res.observation_keys(response_name)
+
```

Comment:
Would it be possible to add the metadata by extending the functionality in the already used `get_response_names()`? Potentially we are looping over all response names in all summarykeys, gendatakeys and genkw keys here for every `response_name` to get the type - and we have already made that list by adding the keys from specific functions? 
```
def get_response_names():
    res = get_res()
    result = res.get_summary_keys().copy()
    result.extend(res.get_gen_data_keys().copy())
    return result
```